Tests/SonarScanner.MSBuild.Common.Test/ProcessRunnerTests.cs

```diff
@@ -633,7 +633,9 @@ private static string ScriptInit() =>
 
     private static string LogArgsPath()
     {
-        var basePath = Path.GetDirectoryName(typeof(ProcessRunnerTests).Assembly.Location).Replace("SonarScanner.MSBuild.Common.Test", "LogArgs");
+        var basePath = Path.GetDirectoryName(typeof(ProcessRunnerTests).Assembly.Location)
+            .Replace("__Instrumented_SonarScanner.MSBuild.Common.Test", null)  // AltCover adds one more folder
+            .Replace("SonarScanner.MSBuild.Common.Test", "LogArgs");
```

Comment:
I don't understand this change and the test does not fail (locally) if I revert it.